src/main/java/application/App.java

```diff
@@ -65,19 +67,74 @@ private static Router createRouter(String directory) {
             String uniqueRoute = app.getUniqueRoute(request.getRoute());
             String resourceRoute = app.saveResource(uniqueRoute, request.getContentFileType(),
                     request.getBody());
-            response.successfulPost(resourceRoute);
+            response.forPost(resourceRoute);
         });
 
         app.put("/cat/1", (Request request, Response response) -> {
             app.saveResource(request.getRoute(), request.getContentFileType(), request.getBody());
-            response.successfulPut();
+            response.forPut();
         });
 
         app.get("/multiple_parameters", (Request request, Response response) -> {
             String body = "Parameters: \n" + request.getParameters().entrySet();
-            response.sendBody(body, MIMETypes.plain);
+            response.setBody(body, MIMETypes.plain);
+        });
+
+        app.get("/form", (Request request, Response response) -> {
+            HTMLBuilder htmlBuilder = new HTMLBuilder();
+            htmlBuilder.append("<h2>Registration</h2>\n" +
+                    "\n" +
+                    "<form method=\"get\" action=\"/form_action\">\n" +
```

Comment:
the form body I feel like should be extracted out somewhere and the form_action route be interpolated in somehow—like what if you change the route name?